server/plugin.go

```diff
@@ -498,6 +498,7 @@ func (p *Plugin) onActivate() error {
 		if err = p.API.UnregisterPluginForSharedChannels(pluginID); err != nil {
 			p.API.LogWarn("Unable to unregister plugin for shared channels", "error", err)
 		}
+		p.API.LogInfo("Unregistering plugin for shared channels since sync msg disabled")
```

Comment:
nit: can we put this before we make the call? (If it fails and we log a warning, the logs won't quite make sense.)

Reply:
:+1: